src/systemTest/java/randoop/main/RandoopSystemTest.java

```diff
@@ -71,6 +71,16 @@ public static void setupClass() {
         SystemTestEnvironmentManager.createSystemTestEnvironmentManager(classpath, buildDir);
   }
 
+  /**
+   * Sets common system test command line options.
+   *
+   * @param options RandoopOptions to set
+   */
+  public static void setSystemTestOptions(RandoopOptions options) {
+    options.setOption("side-effect-free-methods", "resources/systemTest/JDK-sfe-methods.txt");
```

Comment:
Aren't these the default?  Why do they have to be specified?  And why are they in resources/systemTest/ rather than resources/ ?

Reply:
These are the system-test specific ones (java7.util7.*). Perhaps a better file name is warranted.

Reply:
Thanks for the explanation.  Changing `JDK` to `JDK7` in the file name would be adequate, I think.